editor/frontend/src/EditTool.ts

```diff
@@ -57,9 +63,10 @@ export class EditTools {
     this._initialized = true;
 
     const i18n = IModelApp.i18n.registerNamespace(this.namespace);
+    const registerAllTools = undefined !== options?.registerAllTools;
 
     // Register requested tools...
-    if (undefined !== options?.registerUndoRedoTools) {
+    if (registerAllTools || undefined !== options?.registerUndoRedoTools) {

```

Comment:
actually, i think this should just be:
```ts
   const registerAllTools = options?.registerAllTools;
    if (registerAllTools || options?.registerUndoRedoTools) 
```
right? If options.regesterUndoTools is `false` you don't want to do it

Reply:
Can't be false, only be true or undefined.

Reply:
yes, that's what the typescript definition says, but in JavaScript it can be anything. I think it's clearer to not have the explicit `undefined !==`

Reply:
Fine, changed.